code/particle/ParticleSource.h

```diff
@@ -71,7 +71,7 @@ class SourceOrigin {
 	 * 
 	 * @param posOut The pointer where the location will be stored
```

Comment:
Don't forget to document the changes here too

Reply:
All right, taken care of!